packages/studio-be/src/studio/cloud/cloud-router.ts

```diff
@@ -9,6 +9,7 @@ import { StudioServices } from 'studio/studio-router'
 import { CustomStudioRouter } from 'studio/utils/custom-studio-router'
 
 const MAX_BODY_CLOUD_BOT_SIZE = 100 * 1024 * 1024 // 100 MB
+const TOO_LARGE_MESSAGE = 'Chatbot is too large to be uploaded on Botpress cloud'
```

Comment:
Translation?

Reply:
That's an error message, we never translate them